frontend/src/components/SearchBar/FilterBar.test.tsx

```diff
@@ -7,7 +7,7 @@ const componentRender = () => {
   let component = create(
     <div>
       <FilterBar
-        initialValues={{ businessIdentifier: 'test', firstName: 'user' }}
```

Comment:
All FE changes are related to leaky abstractions from the backend breaking frontend logic.